docs/source/config_files.rst

```diff
@@ -119,6 +119,20 @@ An example usecase of ``postBuild`` file is JupyterLab's demo on mybinder.org.
 It uses a ``postBuild`` file in a folder called ``binder`` to `prepare
 their demo for binder <https://github.com/jupyterlab/jupyterlab-demo/blob/master/binder/postBuild>`_.
 
+.. _start:
+
+``start``
```

Comment:
How do folks feel about the name "start"? It seems reasonable to me, but I'm wondering if anybody knows of a similar pattern that other communities have used? (similar to how we've used Heroku as inspiration a few times)?

(also either way, can you make sure the number of `^^^^` below matches the length of the title :-) )